Tests/SentryTests/UIRedactBuilderTests.swift

```diff
@@ -0,0 +1,130 @@
+#if os(iOS)
+import Foundation
+import Nimble
+@testable import Sentry
+import UIKit
+import XCTest
+
+class UIRedactBuilderTests: XCTestCase {
+    
+    private class RedactOptions: SentryRedactOptions {
+        var redactAllText: Bool = true
+        var redactAllImages: Bool = true
+    }
```

Comment:
`m`: If I'm not mistaken we always test with both values true. It would be great to test with different combinations of these too.

Reply:
True. Forgot to test no redacting when the flag is false